scripts/ligandtocore/QD.py

```diff
@@ -0,0 +1,216 @@
+import copy
+import os
+import itertools
+import time
+
+from scm.plams import *
```

Comment:
@BvB93 the `from scm.plams import *` is not a good programming practice. Import only the functionality that you are using